packages/webapp/src/containers/GoogleLoginButton/saga.js

```diff
@@ -0,0 +1,38 @@
+import { createAction } from '@reduxjs/toolkit';
+import { put, takeLatest, call, select } from 'redux-saga/effects';
+import { url } from '../../apiConfig';
+import { onLoadingUserFarmsStart, onLoadingUserFarmsFail } from '../userFarmSlice';
+import history from '../../history';
+import { loginSelector, loginSuccess } from '../loginSlice';
+import { toastr } from 'react-redux-toastr';
+
+const axios = require('axios');
+const loginUrl = () => `${url}/login/google`;
+
+export const loginWithGoogle = createAction(`loginWithGoogleSaga`);
+
+export function* loginWithGoogleSaga({ payload: google_id_token }) {
+  try {
+    yield put(onLoadingUserFarmsStart());
+    const header = {
+      headers: {
+        'Content-Type': 'application/json',
+        'Authorization': 'Bearer ' + google_id_token,
+      },
+    };
+    const result = yield call(axios.post, loginUrl(), {}, header);
```

Comment:
We should use `packages/webapp/src/apiConfig.js` to keep things consistent. If not in this line specifically, it can be used at (or before) the function declaration.